tests/storage/overlay-layers/layers.json

```diff
@@ -0,0 +1 @@
+[{"id": "994393dc58e7931862558d06e46aa2bb17487044f670f310dffe1d24e4d1eec7", "created": "2022-09-25T21:32:34.23624767Z", "compressed-diff-digest": "sha256:213ec9aee27d8be045c6a92b7eac22c9a64b44558193775a1a7f626352392b49", "compressed-size": 2806054, "diff-digest": "sha256:994393dc58e7931862558d06e46aa2bb17487044f670f310dffe1d24e4d1eec7", "diff-size": 5826560, "compression": 2, "uidset": [0], "gidset": [0, 42]}]
```

Comment:
I think you probably want to remove this file

Reply:
This is used in a test